source/arm9/math.c

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
Remove the empty line on top, and follow the pattern of the other files (leave the empty comment in the middle with just "//". Also, leave an empty line after the comments and before the include.